creator-node/src/ipfsClient.js

```diff
@@ -22,4 +26,74 @@ async function logIpfsPeerIds () {
   logger.info(`Current IPFS Peer ID (using latest version of ipfs client): ${JSON.stringify(identityLatest, null, 2)}`)
 }
 
-module.exports = { ipfs, ipfsLatest, logIpfsPeerIds }
+/**
+ * Wrapper to ipfs.addToFs() -- This wrapper allows for ipfs calls to be made async or synchronous.
+ * @param {Object} ipfs ipfs library -- can be two different versions (see ipfsClient.js)
+ * @param {string|string[]} buffers
+ * @param {Object?} config ipfs add config options
+ * @param {string?} caller the name of the caller fn
+ * @param {boolean?} isMetadata if the input data is of a metadata type. if so, always synchronously add to ipfs
+ * @returns {string|string[]} hashes
+ */
+async function ipfsAddWrapper (ipfs, buffers, config = {}, caller = '', isMetadata = false) {
+  // Either an array of hashes, or a hash
+  // Note: the if case is specifically written for the `ipfsLatest` add api. See ipfs add code in `resizeImage.js`
+  let onlyHashes = []
+  let ipfsDaemonHashes
+  if (Array.isArray(buffers)) {
+    for (const { content } of buffers) {
+      const hash = await Hash.of(content)
```

Comment:
do we need a cidVersion: 0 here?

Reply:
added cidVersion: 0 into our custom content hashing logic instead :)